lib/inferno/apps/cli/execute.rb

```diff
@@ -44,10 +44,12 @@ def run(options)
 
         self.options = options
 
-        outputter.print_start_message(options)
+        outputter.print_start_message(self.options)
```

Comment:
This is fine, but you shouldn't need to say `self.options` rather than just `options` unless you are reassigning it since they both refer to the same object.